### PR TITLE
Document Jest light runner in migration guide

### PR DESCRIPTION
**Pull request by jeddy3**
> Which issue, if any, is this issue related to?

Ref https://github.com/stylelint/jest-preset-stylelint/pull/87#discussion_r1403790643

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: docs/migration-guide/to-16.md [hunk 1]
```diff
@@ -154,6 +154,16 @@ We've published a new version of [Jest preset](https://github.com/stylelint/jest
 }
 ```
 
+If you get a segmentation fault while running the preset on Node.js 18, you can use [jest-light-runner](https://www.npmjs.com/package/jest-light-runner):
+
+```diff json
+export default {
```
Comment (ybiquitous) [inline docs/migration-guide/to-16.md:160]: [ask] Is `diff json` incorrect? The snippet seems JS code.

Comment (jeddy3) [reply docs/migration-guide/to-16.md:160]: Oops, good catch.

File: docs/migration-guide/to-16.md [hunk 2]
```diff
@@ -154,6 +154,16 @@ We've published a new version of [Jest preset](https://github.com/stylelint/jest
 }
 ```
 
+If you get a segmentation fault while running the preset on Node.js 18, you can use [jest-light-runner](https://www.npmjs.com/package/jest-light-runner):
+
+```diff js
+export default {
+  preset: "jest-preset-stylelint",
+  setupFiles: ["./jest.setup.js"],
++"runner": "jest-light-runner",
```
Comment (ybiquitous) [inline docs/migration-guide/to-16.md:163]: [suggest] Just for code style consistency:

```suggestion
+ runner: "jest-light-runner",
```

**Review by ybiquitous — APPROVED** [review]
Thank you. LGTM 👍🏼 

I've left a minor suggestion, but you can merge it as-is.

**Pull request merged**